clients/src/main/java/org/apache/kafka/server/authorizer/AclDeleteResult.java

```diff
@@ -44,8 +46,8 @@ private AclDeleteResult(Collection<AclBindingDeleteResult> deleteResults, ApiExc
     /**
      * Returns any exception while attempting to match ACL filter to delete ACLs.
```

Comment:
Should we update this javadoc in the same way you did the `AclCreateResult` one?